packages/mdc-textfield/helper-text/README.md

```diff
@@ -76,9 +76,9 @@ CSS Class | Description
 
 Mixin | Description
 --- | ---
-`mdc-text-field-helper-text-color($color)` | Customizes the color of the helper text following an enabled text-field.
-`mdc-text-field-disabled-helper-text-color($color)` | Customizes the color of the helper text following a disabled text-field.
-`mdc-text-field-helper-text-validation-color($color)` | Customizes the color of the helper text when it's used as a validation message.
+`helper-text.helper-text-color($color)` | Customizes the color of the helper text following an enabled text-field.
```

Comment:
Thanks for the PR! To follow convention with other readmes, can you remove the `helper-text.` prefix? A Sass module's namespace is configurable by the user, so it may not always be "helper-text"